File: loxgo/lox/fibrec.js
const fib = (n) => {
  if (n <= 1) return n;
  return fib(n - 2) + fib(n - 1);
};

const start = new Date().getTime();
for (let i = 0; i < 30; i = i + 1) {
  console.log(fib(i));
}
const end = new Date().getTime();

console.log("TIME");
console.log((end - start) / 1000);

File: loxgo/lox/fibrec.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 2) + fib(n - 1)

start = time.time()
for i in range(0,30):
    print(fib(i))
end = time.time()

print("TIME")
print((end - start))